prov/util/src/util_mr_map.c

```diff
@@ -108,22 +108,37 @@ int ofi_mr_map_verify(struct ofi_mr_map *map, uintptr_t *io_addr,
 	void *addr;
 
 	node = ofi_rbmap_find(map->rbtree, &key);
-	if (!node)
-		return -FI_EINVAL;
+	if (!node) {
+                FI_WARN(map->prov, FI_LOG_MR,
+                    "verify_addr: unknown key: %" PRIu64 "\n", key);
```

Comment:
Please align with the "m" in the previous line. Same for other places.

Reply:
Fixed.